src/workers/python/init.py.ts

```diff
@@ -1,44 +1,140 @@
+export const INITIALIZE_PYTHON_BACKEND = "__init_papyros";
+export const PROCESS_PYTHON_CODE = "__process_code";
 export const INITIALIZATION_CODE =
     `
-from pyodide import to_js
+from pyodide import to_js, eval_code_async
+from js import console
 import sys
+import ast
+import json
+import html
+import types
+import os
+import micropip
+await micropip.install('friendly_traceback')
+import friendly_traceback
+from friendly_traceback.core import FriendlyTraceback
```

Comment:
Both installing and simply importing friendly_traceback affect startup significantly. Try starting the installation here but only awaiting the installation, importing, and updating the source cache when an exception is raised.